resources/components/base/Icon.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+	<span
+		class="wbmad-icon"
+		v-bind:class="classArray"
+		v-bind:title="title"
+	/>
+</template>
+
+<script>
+module.exports = {
+	name: 'Icon',
+	props: {
+		icon: {
+			type: String,
+			default: null
+		},
+		invert: {
+			type: Boolean,
+			default: false
+		},
+		title: {
+			type: String,
+			default: null
+		}
+	},
+
+	computed: {
+		classArray() {
```

Comment:
I think we should always return class lists as objects for consistency.

Reply:
Updated!